extensions/wikia/CuratedContent/maintenance/HeroImageToCuratedContentMigrator.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '/../../../../maintenance/Maintenance.php' );
+
+//Migrates image and description from hero module to curated content.
+
+class HeroImageToCuratedContentMigrator extends Maintenance {
+
+	function execute() {
+		global $wgCityId;
+		$commData = new CommunityDataService($wgCityId);
+		$curatedData = $commData->getCuratedContentData();
+
+		$wikiData = new WikiDataModel(Title::newMainPage()->getText());
+		$wikiData->getFromProps();
+
+		// migrate description if there is no description in community data
+		if (empty($commData->getCommunityDescription()) && !empty($wikiData->description)) {
+			$curatedData['community_data']['description'] = $wikiData->description;
+		}
+
+		$heroImage = Title::newFromText($wikiData->getImageName(), NS_IMAGE);
```

Comment:
Is NS_IMAGE a key namespace or an alias to NS_FILE ? If so, then please use the 'canonical' one :)